scripts/api_Restaurant.js

```diff
@@ -43,12 +43,12 @@ function AddFoodComponentToDom(nameOfRestaurant) {
 function createFoodSearchComponent(foodObj) {
     return `
         <h2 id = rest-${uniqueRestId}>${foodObj.restaurant.name}</h2>
-            <button id = butt-${uniqueButtonId}>Save</button>
+            <button id = butt-${uniqueRestButtonId}>Save</button>
     `;
 }
 
 function putMyRestaurantIntoTheItinerary (restaurantElement) {
-    document.getElementById("restaurant-itinerary").innerHTML = restaurantElement;
+    document.getElementById("restaurant-itinerary").innerHTML = "Restaurant: " + restaurantElement;
```

Comment:
Yeah we just need to keep an eye on how this puppy is going to behave vs. the others. And we need to remember to inspect to make sure the HTML structures are identical so we don't end up with styling anomolies!

Reply:
i gotcha, I wasnt sure at first what you were getting at cause im crap at styling but i think i understand now!

Reply:
let me merge and if you want to take that part out for a smoother styling you have my blessing, I just want to make sure its working for everyone first